resources/views/users/view.blade.php

```diff
@@ -343,6 +343,30 @@
                         </div>
                       </div>
 
+                   <!-- start date -->
+                   @if ($user->start_date)
+                   <div class="row">
```

Comment:
Tiny teensy microscopic nit - can we indent the innards of the `@if` directive?

Reply:
I already merged that, but I did apply that indent directly to develop